doc/fr/api/clapi/objects/instances.rst

```diff
@@ -129,6 +129,8 @@ nagios_bin                 Path of the Scheduler binary
 
 nagiostats_bin             Path of the Nagios Stats binary
 
+ssh_port                   SSH Port
+
```

Comment:
shouldn't the SSH port option, be after the gorgone_communication_type ?

Reply:
Actually it's the legacy port used until Gorgone, but nor by Gorgone, so different from gorgone_port, so before gorgone_communication_type .

Reply:
but in gorgone_communication_type you need to specify if you use the ZMQ or the SSH ;)

Reply:
I know but SSH communication type also refer to gorgone_port.
ssh_port is not used by Gorgone.